examples/jsm/renderers/webgpu/WebGPURenderPipelines.js

```diff
@@ -71,7 +53,8 @@ class WebGPURenderPipelines {
 
 				moduleVertex = {
 					module: device.createShaderModule( { code: byteCodeVertex } ),
-					entryPoint: 'main'
+					entryPoint: 'main',
+					id: _id ++
```

Comment:
@Kangz Do you think it's okay to enhance a descriptor with custom data? `GPUProgrammableStage` does not have an `id` property however it's useful for having it at that place (or the shader module) when computing a cache key.

Reply:
Yeah it's fine. It will be ignore by Chromium.

Reply:
@Mugen87 Should we add a comment to make clear it's custom data?

Reply:
This has already been refactored. The id is now a property of `WebGPUProgrammableStage` which encapsulates the above data. 👍